db/primary_migrate/20230502235856_remove_profile_fraud_boolean_columns.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveProfileFraudBooleanColumns < ActiveRecord::Migration[7.0]
+  def change
+    safety_assured {
```

Comment:
```suggestion
    remove_index :profiles, :fraud_review_pending
    safety_assured {
```

A more helpful suggestion than the one below. I messed up the name, it's `remove_index` rather than `drop_index`